src/selectors/minColWidth.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule minColWidth
+ */
+import WidthHelper from 'FixedDataTableWidthHelper';
+import { createSelector } from 'reselect';
+
+/**
+ * TODO (jordan) Check if this is cached incorrectly
```

Comment:
I should follow up on this...

Reply:
Tested this and it's fine since adding rows would pass props in and reset the columns.  I'm not sure if rowHeightGetter would cause trouble since we should know if we're scrolling or not before we every compute the flex widths.

Still all fix it in a follow up just in case.